MIGRATING.md

```diff
@@ -36,26 +48,88 @@ Follow these steps to remove the tracing library and its dependencies:
+#### OTel equivalents of SignalFx instrumentation gems
+
+| SignalFx instrumentation name | OTel instrumentation name | notes |
+| ----------------------------- | ------------------------- | ----- |
+| signalfx-activerecord-opentracing | opentelemetry-instrumentation-active_record | |
+| signalfx-elasticsearch-instrumentation | no known equivalents | [open issue in OTel](https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/8) |
+| signalfx-faraday-instrumentation       | opentelemetry-instrumentation-faraday | |
+| signalfx-grape-instrumentation         | no known equivalents | [open issue in OTel](https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/9) |
+| signalfgx-mongodb-instrumentation      | opentelemetry-instrumentation-mongo | |
+| signalfx-mysql2-instrumentation        | opentelemetry-instrumentation-mysql2 | |
+| signalfx-nethttp-instrumentation       | opentelemetry-instrumentation-net_http | |
+| signalfx-pg-instrumentation            | opentelemetry-instrumentation-pg | |
+| signalfx-rack-tracer                   | opentelemetry-instrumentation-rack | |
+| signalfx-rails-instrumentation         | opentelemetry-instrumentation-rails | |
+| signalfx-redis-instrumentation         | opentelemetry-instrumentation-redis | |
+| signalfx-restclient-instrumentation    | opentelemetry-instrumentation-restclient | |
+| signalfx-sequel-instrumentation        | no known equivalents | [open issue in OTel](https://github.com/open-telemetry/opentelemetry-ruby-contrib/issues/11) |
+| signalfx-sidekiq-opentracing           | opentelemetry-instrumentation-sidekiq | |
+| signalfx-sinatra-instrumentation       | opentelemetry-instrumentation-sinatra | |
+
+#### OTel equivalents of other Open Tracing instrumentation gems
+
+See the list of all known OTel Ruby instrumentations: <https://opentelemetry.io/registry/?language=ruby&component=instrumentation>
+
 ### Migrate settings for the Splunk Ruby OTel agent
 
-To migrate settings from the SignalFx tracing library to the Splunk
-Distribution of OpenTelemetry Ruby, rename the following environment
-variables:
+1. `SIGNALFX_ENDPOINT_URL` or `ingest_url` configuration parameter
+    - if you installed Smart Agent and can migrate to OpenTelemetry Collector, do that first, then see the point below
+    - if you have OpenTelemetry Collector available up as a sidecar (via `localhost`),
+      and it accepts OTLP on default ports, then just remove this setting, we export OTLP to OTel Collector by default
+    - if you need to keep using Smart Agent, then you need to set up a jaeger exporter yourself
+    - if you export directly to our backend (without OTel Collector as a proxy),
+      then just set `SPLUNK_REALM` to your realm
+
+1. `SIGNALFX_SERVICE_NAME` or `service_name` configuration parameter
+    - to set in environment use `OTEL_SERVICE_NAME`
+    - to set in code use `Splunk::Otel.configure(service_name: 'your service name', ...`
+
```

Comment:
to set in code, use `Splunk::Otel.configure(service_name: 'your service name', ...`